school/student.rb

```diff
@@ -1,5 +1,7 @@
 require_relative 'person'
 class Student < Person
```

Comment:
- [ ] Kindly set up the other end of the classroom association in this student class by adding its method.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```